core/src/Kokkos_Complex.hpp

```diff
@@ -798,25 +800,25 @@ template <class RealType>
 KOKKOS_INLINE_FUNCTION Kokkos::complex<RealType> sqrt(
     const complex<RealType>& x) {
 #ifdef KOKKOS_ACTIVE_EXECUTION_MEMORY_SPACE_SYCL
-  using sycl::abs;
+  using sycl::fabs;
   using sycl::sqrt;
 #else
-  using std::abs;
+  using std::fabs;
   using std::sqrt;
 #endif
```

Comment:
Shouldn't this just be
```
  using Kokkos::Experimental::fabs;
  using Kokkos::Experimental::sqrt;
```
?

Reply:
Fixing in upcoming PR that will pull all the `#ifdef KOKKOS_ACTIVE_EXECUTION_MEMORY_SPACE_SYCL`